sympy/physics/control/lti.py

```diff
@@ -20,8 +20,42 @@ def _roots(poly, var):
         r = [rootof(poly, var, k) for k in range(n)]
     return r
 
+class LTICommon:
+    """A common class for all the Linear Time-Invariant Dynamical Systems."""
+    # Users should not directly interact with this class.
+    def __new__(cls, system, **kwargs):
+        if cls is LTICommon:
+            raise NotImplementedError('The LTICommon class is not meant to be used directly.')
+        obj = super(LTICommon, cls).__new__(cls)
+        obj.__var = system.var
+        obj.__num_inputs = getattr(system, 'num_inputs', None)
+        obj.__num_outputs = getattr(system, 'num_outputs', None)
+        return obj
+
+    @property
+    def is_SISO(self):
+        """Returns `True` if the passed LTI system is SISO else returns False."""
+        return self._is_SISO
+
+    def _to_expr(self):
+        """Returns the equivalent ``Expr`` object."""
+        if not self.is_SISO:
+            raise TypeError("`to_expr` method is only valid for SISO Systems.")
+
+        if isinstance(self, Series):
+            return Mul(*(arg.to_expr() for arg in self.args), evaluate=False)
+        elif isinstance(self, Parallel):
+            return Add(*(arg.to_expr() for arg in self.args), evaluate=False)
+        elif isinstance(self, TransferFunction):
```

Comment:
Special-casing on subclasses is not really how these things usually work: just implement this method on each subclass.